apps/events-helsinki/src/domain/event/eventCard/__tests__/EventCard.test.tsx

```diff
@@ -1,4 +1,5 @@
 import type { EventFieldsFragment } from '@events-helsinki/components';
+import { EventCard } from '@events-helsinki/components';
```

Comment:
If the EventCard is now inside the `@events-helsinki/components`, it should not be tested in the application, but in the components-package!

All the moved components should be tested where they are originated! There is no need to test it multiple times in every apps, because it could and should be tested once in the dependency-package.

Reply:
There are same kind of issues in many of the moved files:

`yarn typecheck`
```
Found 19 errors in 17 files.

Errors  Files
     1  src/domain/event/__tests__/EventPageContainer.test.tsx:28
     1  src/domain/event/eventCard/__tests__/EventCard.test.tsx:2
     1  src/domain/event/eventCard/__tests__/LargeEventCard.test.tsx:12
     1  src/domain/event/eventClosedHero/__tests__/EventClosedHero.test.tsx:5
     1  src/domain/event/eventContent/__tests__/EventContent.test.tsx:7
     2  src/domain/event/eventHero/__tests__/EventHero.test.tsx:19
     2  src/domain/event/eventInfo/__tests__/EventInfo.test.tsx:22
     1  src/domain/event/eventInfo/__tests__/OtherEventTimes.test.tsx:29
     1  src/domain/event/eventKeywords/__tests__/EventKeywords.test.tsx:12
     1  src/domain/event/eventName/__tests__/EventName.test.tsx:7
     1  src/domain/event/eventPageMeta/__tests__/EventPageMeta.test.tsx:6
     1  src/domain/event/similarEvents/__tests__/SimilarEvents.test.tsx:18
     1  ../../packages/components/src/components/event/eventCard/EventCard.tsx:7
     1  ../../packages/components/src/components/event/eventCard/LargeEventCard.tsx:13
     1  ../../packages/components/src/components/event/eventHeroClosed/EventClosedHero.tsx:5
     1  ../../packages/components/src/components/event/eventInfo/EventsHierarchy.tsx:13
     1  ../../packages/components/src/components/event/eventInfo/OrganizationInfo.tsx:7
```

Reply:
Some fixes included in https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/376